library/image_streamer_artifact_bundle.py

```diff
@@ -15,59 +15,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ###
-from ansible.module_utils.basic import *
-import os.path
 
-try:
-    from hpOneView.oneview_client import OneViewClient
-    from hpOneView.extras.comparators import resource_compare
-    from hpOneView.exceptions import HPOneViewException
-
-    HAS_HPE_ONEVIEW = True
-except ImportError:
-    HAS_HPE_ONEVIEW = False
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'curated',
+                    'metadata_version': '1.0'}
 
 DOCUMENTATION = '''
 ---
 module: image_streamer_artifact_bundle
 short_description: Manage the Artifact Bundle resource.
 description:
     - "Provides an interface to manage the Artifact Bundle. Can create, update, remove, and download, upload, extract"
+version_added: "2.3"
 requirements:
     - "python >= 2.7.9"
     - "hpOneView >= 3.1.0"
 author:
     - "Abilio Parada (@abiliogp)"
 options:
-    config:
-      description:
-        - Path to a .json configuration file containing the OneView client configuration.
-          The configuration file is optional. If the file path is not provided, the configuration will be loaded from
-          environment variables.
-      required: false
     state:
       description:
         - Indicates the desired state for the Artifact Bundle resource.
-          'present' will ensure data properties are compliant with OneView.
-          'absent' will remove the resource from OneView, if it exists.
-          'downloaded' will download the Artifact Bundle to the file path provided.
-          'archive_downloaded' will download the Artifact Bundle archive to the file path provided.
-          'backup_uploaded' will upload the Backup for the Artifact Bundle from the file path provided.
-          'backup_created' will create a Backup for the Artifact Bundle.
-          'extracted' will extract an Artifact Bundle.
-          'backup_extracted' will extract an Artifact Bundle from the Backup.
+          C(present) will ensure data properties are compliant with OneView.
```

Comment:
Since only the name of the resource can be updated, it could be documented.
Maybe something like:

'When the artifact bundle already exists, only the name is updated. Changes in any other attribute value is ignored'